oxd-server/src/main/java/org/gluu/oxd/server/op/GetAuthorizationUrlOperation.java

```diff
@@ -101,4 +107,24 @@ public IOpResponse execute(GetAuthorizationUrlParams params) throws Exception {
             return new ArrayList<>();
         }
     }
+
+    private void persistRp(Rp rp, UpdateSiteParams params) {
```

Comment:
We don't want to mess together UpdateSite and GetAuthorizationUrl functionalities. Please remove this code.